_posts/2020-03-31-Flyer-Factsheet.md

```diff
@@ -0,0 +1,10 @@
+---
+site: freiburg
+title: Galaxy Europe Flyer and Factsheet for download
+tags: [de.NBI, ELIXIR]
+---
+
+Here we present the latest [Galaxy flyer](/assets/media/GalaxyEurope_Flyer.pdf) with information about our service and the recent Galaxy factsheet. A big "THANK YOU" to all our users for using Galaxy!
```

Comment:
Can we add more info's, like the branding repository (https://github.com/usegalaxy-eu/branding). That people can ask us if they need stickers, or printed versions for their institutes ...

Reply:
@stephanflemming